### PR TITLE
Adding the Comet, a ship intended to be used for light mining and construction/repair jobs.

### PR DESCRIPTION
**Pull request by Shodhans**
…struction/repairs.

## About the PR
 Made a ship that attempts to fill a niche that others don't, enabling easy construction jobs via a slim docking port and easy EVA maneuverability, as well as a suite of tools and some starting resources to help it along. It is priced at $85,500 due to it being larger than a Barge ($42,300) and smaller than a Praeda ($160,210) while fulfilling a similar role to both, while having far less versatility than the Praeda.

## Why / Balance
 I feel like the niche of construction ships is largely unexplored, due to most people being willing to do their own construction and repairs at the cost of time. I feel this ship is decently equipped to handle larger jobs that may otherwise require somebody to sell their ship.

## Media
 
- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://youtu.be/TlGiHq1WPu0

## Breaking changes

**Changelog**

Changelog 

Mk I has been upgraded to Mk II

- Reworked visuals of main hold
- Rearranged main hold to include more open space and more convenient placement of machines
- Moved equipment lockers to underutilized atmos room
- Modified weaponry and tools on board
- Fixed disconnected thruster, swapped some thrusters from rear-facing to side-facing, added a few forward-facing as well to overall reduce forward acceleration but improve maneuverability overall
- Added decal hinting towards location of secret door
- Added new set of airlocks behind docking ports so it didn't feel quite so 'empty'
- New walkthrough video - https://youtu.be/KcHl6yYuVD0

### CODE REVIEW

**Comment by dvir001** [comment]
Please do not use capitals in names, keep the format the same as every other file.

**Comment by IData-RedactedI** [comment]
The  insde seems a little barren, could do with a little decal work just to polish it up. Putting a set of doors between the docking ports and the rest of the ship would also be pretty good, like on the praeda

**Comment by Shodhans** [comment]
> Please do not use capitals in names, keep the file naming format the same as every other file.
> 
> Gun safe has Armory lock, how is anyone going to open it, did you force remove the lock from the entity?
> 
> Ship having the price 2 times whats its worth to sell isn't too great, as it means a 40k base investment on it to start with.
> 
> Base Profile Screenshot 2023 10 27 - 17 12 31 58 Not too sure about the balance of giving guns and security type weapons with the ship like this, but not my call.
> 
> This axe is the syndicate axe also..
> 
> There is thruster that isn't connected to power on the side.
> 
> The air mix a bit off then the normal 22 78, but not an issue.
> 
> Warp point not using the ship kind?

I'll take a look at stuff and fix some things once I get home, thanks for the feedback! Though I am curious what you mean by 'security type weapons' - the only thing that's not available on most ships is the stunbaton, which I did ask for opinions on and told that was probably fine. Fixing the file naming shouldn't be a problem, just didn't realize that it was for an actual reason and not just a coincidence.

Didn't realize I accidentally put a syndi axe, easy fix.

Will re-check thrusters, thought I had fixed them all.

What would you suggest as a price point, then? $60k?

Air mix was set to 21/79 as I was suggest to do so, another very easy fix over

And lastly I'll have to re-check the warp point, I think I actually did wind up putting the correct one in prior to a little snafu with my file that caused me to lose a little bit of work and forgot to swap it back over. As mentioned, will take a look at things once I get home!

**Comment by dvir001** [comment]
> > Please do not use capitals in names, keep the file naming format the same as every other file.
> > Gun safe has Armory lock, how is anyone going to open it, did you force remove the lock from the entity?
> > Ship having the price 2 times whats its worth to sell isn't too great, as it means a 40k base investment on it to start with.
> > Base Profile Screenshot 2023 10 27 - 17 12 31 58 Not too sure about the balance of giving guns and security type weapons with the ship like this, but not my call.
> > This axe is the syndicate axe also..
> > There is thruster that isn't connected to power on the side.
> > The air mix a bit off then the normal 22 78, but not an issue.
> > Warp point not using the ship kind?
> 
> I'll take a look at stuff and fix some things once I get home, thanks for the feedback! Though I am curious what you mean by 'security type weapons' - the only thing that's not available on most ships is the stunbaton, which I did ask for opinions on and told that was probably fine. Fixing the file naming shouldn't be a problem, just didn't realize that it was for an actual reason and not just a coincidence.
> 
> Didn't realize I accidentally put a syndi axe, easy fix.
> 
> Will re-check thrusters, thought I had fixed them all.
> 
> What would you suggest as a price point, then? $60k?
> 
> Air mix was set to 21/79 as I was suggest to do so, another very easy fix over
> 
> And lastly I'll have to re-check the warp point, I think I actually did wind up putting the correct one in prior to a little snafu with my file that caused me to lose a little bit of work and forgot to swap it back over. As mentioned, will take a look at things once I get home!

60k ~

**Comment by Shodhans** [comment]
> The insde seems a little barren, could do with a little decal work just to polish it up. Putting a set of doors between the docking ports and the rest of the ship would also be pretty good, like on the praeda

I was sitting in the decal menu for a hot second trying to figure out what I could do to spruce it up, but that's not my strong point. Will take another pass when I go work through Divir's feedback later tonight. Good idea on the doors.

**Comment by Shodhans** [comment]
Hopefully a better run this time, including a new visual preview

https://youtu.be/KcHl6yYuVD0

**Comment by Cheackraze** [comment]
[x] fits a niche
[x] theme makes sense
[x] unique shape and purpose
[x] fully fuctional

I like it! As a few others mentioned, it could use a tad sprucing up with some more decal work. Maybe take a look at how the Barge has a single department themed stripe running along it to help fill in some of the 'blank' areas.  Decal work is the most daunting imo but once you get into a 'flow' and get some practice in, its honestly my favorite part of mapping and its like putting those final decorations on a really fancy cake.

My biggest issue personally is blending normal walls and shuttle walls. For a 'construction ship' I would actually recommend avoiding the Shuttle themed walls for now, and instead replace them with Reinforced Walls and Reinforced Windows.  This will allow players to customize the ship itself (this looks like an incredible ship for it tbh) and have it blend perfectly.  It will also allow you to use normal walls in the interior, along with your hidden door.

Im excited to get this in game! Keep up the good work, its almost there!

File: SpaceStation14.sln
Comment (Cheackraze) [inline SpaceStation14.sln:1]: gotta not include this one in your commits

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**